USAGE.md

```diff
@@ -111,10 +115,10 @@ If you are on a windows machine ensure that you open powershell in administrator
 
 This will add an entry for the document manager backend if it does not currently exist.
 
-On a mac or linux run the following:
+On a mac or linux run the following in the bin directory:
 
 ```
-./AddHosts.sh add 127.0.0.1 document_manager_backend
+sudo ./AddHosts.sh add 127.0.0.1 document_manager_backend
```

Comment:
add this to the make commands

Reply:
For now I did not do this because of the whole admin access required to run these, but they're easier to find now